src/module/vue/components/character-sheet-tabs/sf-connections.vue

```diff
@@ -13,7 +13,7 @@
           @sortUp="sortUp"
           @sortDown="sortDown"
         />
-        <progress-box :item="item" :show-star="true" />
+        <progress-list-item :actor="actor" :item="item" :show-star="true" />
```

Comment:
With the other suggestion, you could drop the prop use everywhere:

```suggestion
        <progress-list-item :item="item" :show-star="true" />
```